### PR TITLE
Expiration Notifications

### PR DESCRIPTION
**Pull request by kobergj**
Send notifications when shares (or space memberships) expire

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>Core-API-Tests-ocis-storage-8</strong>](https://drone.owncloud.com/owncloud/ocis/17953/56/1) failed. Further test are cancelled...

File: services/notifications/pkg/email/templates/shares/shareExpired.email.body.tmpl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Even though this share has been revoked you still might have access through other shares and/or space memberships
+
+----------------------------------------------------------
+
+Hallo {{ .ShareGrantee }},
+
+Deine Freigabe zu {{ .ShareFolder }} ist am {{ .ExpiredAt }} abgelaufen.
+
+Obwohl diese Freigabe nicht mehr zur Verfügung steht könntest du immernoch Zugriff über andere Freigaben und/oder space Mitgliedschaften haben.
```
[the remark this answers is not in the document]
Comment (kobergj) [reply services/notifications/pkg/email/templates/shares/shareExpired.email.body.tmpl]: :+1: Good find!

File: services/notifications/pkg/email/templates/shares/shareExpired.email.body.tmpl [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Even though this share has been revoked you still might have access through other shares and/or space memberships
+
+----------------------------------------------------------
+
+Hallo {{ .ShareGrantee }},
+
+Deine Freigabe zu {{ .ShareFolder }} ist am {{ .ExpiredAt }} abgelaufen.
+
+Obwohl diese Freigabe nicht mehr zur Verfügung steht könntest du immernoch Zugriff über andere Freigaben und/oder Space Mitgliedschaften haben.
```
Comment (micbar) [inline services/notifications/pkg/email/templates/shares/shareExpired.email.body.tmpl:13]:
```suggestion
Obwohl diese Freigabe nicht mehr zur Verfügung steht, könntest du immernoch Zugriff über andere Freigaben und/oder Space Mitgliedschaften haben.
```

File: services/notifications/pkg/email/templates/spaces/membershipExpired.email.body.tmpl [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Even though this membership has expired you still might have access through other shares and/or space memberships
+
+----------------------------------------------------------
+
+Hallo {{ .SpaceGrantee }},
+
+Deine Mitgliedschaft zu dem Space {{ .SpaceName }} ist am {{ .ExpiredAt }} abgelaufen.
+
+Obwohl diese Mitgliedschaft nicht mehr zur Verfügung steht könntest du immernoch Zugriff über andere Freigaben und/oder Space Mitgliedschaften haben.
```
Comment (micbar) [inline services/notifications/pkg/email/templates/spaces/membershipExpired.email.body.tmpl:13]:
```suggestion
Obwohl diese Mitgliedschaft nicht mehr zur Verfügung steht, könntest du immer noch Zugriff über andere Freigaben und/oder Space Mitgliedschaften haben.
```

**Review by micbar — APPROVED** [review]
LGTM

**Pull request merged**